src/themes/light.scss

```diff
@@ -2,16 +2,9 @@
   --wxc-meeting-background: linear-gradient(180deg, #F0F0F0 0%, #FFFFFF 100%);
   --wxc-text-color: rgba(0, 0, 0, 0.95);
   // Icons
```

Comment:
no need for this comment anymore (there are few variables now), but please leave a blank line